app/post.go

```diff
@@ -24,6 +24,12 @@ import (
 	"golang.org/x/net/html/charset"
 )
 
+const (
+	DEFAULT_LIMIT_POSTS_SINCE = 60
+	PAGE_DEFAULT              = 0
+	PER_PAGE_DEFAULT          = 60
```

Comment:
Why not use the existing constants for these?

Reply:
I assume you're referring to those at [`web` package](https://github.com/mattermost/mattermost-server/blob/master/web/params.go#L15-L16).  I tried that but I'm hitting circular dependency error.
The least I could do is putting all those in `app` package but I'm hesitant to do it that way since I thought we're accessing app in api4 via context only.  However, I've other places did that and so with my new update.  Please take a look and let me know if that's good to go.